ui/component/fileDownloadLink/view.jsx

```diff
@@ -102,16 +106,24 @@ function FileDownloadLink(props: Props) {
   const label = __('Download');
 
   return (
-    <Button
-      button={buttonType}
-      className={buttonType ? undefined : 'button--file-action'}
-      title={label}
-      icon={ICONS.DOWNLOAD}
-      label={showLabel ? label : null}
-      onClick={handleDownload}
-      aria-hidden={!focusable}
-      tabIndex={focusable ? 0 : -1}
-    />
+    <>
+      {contentRestrictedFromUser && (
+        <h2 className="protected-download-header">
+          This download is protected content, join a membership to gain access
+        </h2>
+      )}
+      <Button
+        button={buttonType}
+        className={buttonType ? undefined : 'button--file-action'}
+        title={label}
+        icon={ICONS.DOWNLOAD}
+        label={showLabel ? label : null}
+        onClick={handleDownload}
+        aria-hidden={!focusable}
+        tabIndex={focusable ? 0 : -1}
+        disabled={contentRestrictedFromUser}
+      />
+    </>
```

Comment:
Minor.  Rave can fix it later.
- The `h2` here isn't ideal, because the component is used as a child of a section, which already has a `h2`.  This causes 2 `h2`s to look entirely different.
- Suggestion: from a component design and re-usability perspective, putting the text below the button in the regular warning/error style would make fit any parent.

![image](https://user-images.githubusercontent.com/64950861/196736726-644c7d2d-20dd-49ad-b53f-f66af25f4275.png)